functions/snippets/src/main/java/Scopes.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START functions_tips_scopes]
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Arrays;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class Scopes {
+  // [END functions_tips_scopes]
+  private static int lightComputation() {
```

Comment:
Since these functions are excluded from the snippet, this snippet is no longer copy-paste-runnable. If you remove the region tag excludes, you can move these functions below `scopeDemo` so that they are still part of the sample, but the user doesn't have to read them first.

This way the sample is still copy-paste-runnable, but the user doesn't have to read through the functions immediately.

Reply:
None of the languages for `functions_tips_scopes` are copy-paste-runnable.

Do we want to either:
- Keep the sample consistent
- Include these functions but be inconsistent

Reply:
We should include these functions. It's fine for older versions of samples to be grandfathered in, but we should aim to keep newer samples up to date with our guidelines, since they were created with the user in mind.

Reply:
Included the functions.

Reply:
nit: I would still suggest moving them both below `scopeDemo` (since `scopeDemo` is really what you want the user to focus on).

Reply:
Ah. Moved the functions location to the bottom of the class.